crates/wit-component/src/gc.rs

```diff
@@ -707,6 +707,7 @@ impl<'a> Module<'a> {
             num_func_imports += 1;
         }
 
+        let mut realloc_via_memory_grow_index = None;
```

Comment:
I think that the changes might no longer be required in this file? Or are they perhaps related to a bugfix otherwise?

Reply:
Thanks, you are right, reverted.